sda-commons-server-kafka/README.md

```diff
@@ -19,7 +19,7 @@ compile 'org.sdase.commons:sda-commons-server-kafka:<current-version>'
 
 | Group                        | Name                    | Version     | Description                  |
 |------------------------------|-------------------------|-------------|------------------------------|
-| `org.apache.kafka`           | `kafka-clients`         | 1.1.1       | Client API for Apache Kafka  |
+| `org.apache.kafka`           | `kafka-clients`         | 2.2.1       | Client API for Apache Kafka  |
 | `com.github.ftrossbach`      | `club-topicana-core`    | 0.1.0       | Helper for Topic description | 
```

Comment:
Do we need this table? I think that will go out of date to quickly?

Reply:
Not a fan either. I removed that section from the documentation.